packages/components/src/components/keyword/KeywordTag.tsx

```diff
@@ -44,7 +44,8 @@ const KeywordTag: FunctionComponent<Props> = ({
         transparent && styles.transparent,
         className
       )}
-      onClick={handleClick}
+      /* Disable keyword onClick for now per LIIKUNTA-411 comment */
+      /* onClick={handleClick} */
```

Comment:
If the onClick is commented out, the handleClick-function throws eslint warning `'handleClick' is assigned a value but never used.`.

Proposal:
```typescript
  // eslint-disable-next-line @typescript-eslint/no-unused-vars
  const handleClick = (): void => {
    setIsSelected(true);
    onClick && onClick();
  };
```

Reply:
A bit related to to this, there is a ticket https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-438 that is for tags overall look.

Reply:
Will do this